jdbc-perf-logger-driver/src/test/java/ch/sla/jdbcperflogger/driver/WrappingDriverUnloadTest.java

```diff
@@ -15,65 +15,81 @@
  */
 package ch.sla.jdbcperflogger.driver;
 
+import static org.awaitility.Awaitility.await;
+
 import java.io.IOException;
 import java.net.ServerSocket;
 import java.net.Socket;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
 
-import org.junit.Assert;
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
FYI under eclipse, which I'm using, my package-info is annotated with `@NonNullByDefault` so that it is implicit that all methods return @NonNull types. I usually only use `@Nullable` where needed.

Reply:
Sorry, I don't get it. which effect should this import have?

Reply:
Ah, guess it should be removed :-)